app/controllers/home_controller.rb

```diff
@@ -1,4 +1,21 @@
 class HomeController < ApplicationController
   def index
+    if user_signed_in?
+      @post=Post.new
```

Comment:
What is purpose of `@post=Post.new` of this statement

Reply:
Removed
Done